amplitude.js

```diff
@@ -439,6 +444,30 @@ Amplitude.prototype.logEvent = function(eventType, eventProperties) {
   }
 };
 
+Amplitude.prototype.logEvent = function(eventType, eventProperties) {
+  this._logEvent(eventType, eventProperties);
+};
+
+// Test that n is a number or a numeric value.
+var _isNumber = function(n) {
+  return !isNaN(parseFloat(n)) && isFinite(n);
+};
+
+Amplitude.prototype.logRevenue = function(price, quantity, product) {
+  // Test that the parameters are of the right type.
+  if (!_isNumber(price) || quantity !== undefined && !_isNumber(quantity)) {
```

Comment:
@cdll we do in our new documentation for logRevenueV2: https://amplitude.zendesk.com/hc/en-us/articles/115001361248-JavaScript-SDK-Installation#tracking-revenue